camino/solvers/decomposition/sequential_benders_master.py

```diff
@@ -438,14 +444,15 @@ def update_options(self, relaxed=False):
             self.internal_lb = self.y_N_val
 
         if relaxed:
-            # self.options['gurobi.MIPGap'] = 1.0
-            self.options[self.mipgap_options_str] = self.mipgap_miqp
+            self.options[self.mipgap_options_str] = 1.0
```

Comment:
i believe we should remove this and instead use the option `nlp_rel+s-b-miqp` for obtaining the same effect.

Reply:
We still need this part of the code, no? (To avoid getting it flagged as an integer solution). But can be a good improvement to clean it up.

Reply:
I'll open as issue to not forget about it. We can later decide what to do